rosie/federal_senate/adapter.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import numpy as np
+import pandas as pd
+from serenata_toolbox.federal_senate.federal_senate_dataset import FederalSenateDataset
+from serenata_toolbox.datasets import fetch
+
+COLUMNS = {
+    'net_value': 'total_net_value',
+    'recipient_id': 'cnpj_cpf',
+    'recipient': 'supplier',
+}
+
+class Adapter:
+    def __init__(self, path):
+        self.path = path
+
+    @property
+    def dataset(self):
+        path = self.update_datasets()
+        self._dataset = pd.read_csv(path, dtype={'cnpj_cpf': np.str}, encoding = "utf-8")
+        self.prepare_dataset()
+
+        return self._dataset
+
+    def prepare_dataset(self):
+        self.prepare_cpnj_cpf()
+        self.rename_columns()
+
+    def prepare_cpnj_cpf(self):
+        self._dataset = self._dataset[self._dataset['cnpj_cpf'].notnull()]
+        self._dataset['document_type'] = 'simple_receipt'
```

Comment:
Yes, it is a padronization for the core module :) 
To run `invalid_cpnj_cpf` the dataset must have this field.

I can comment it on the code!